lib/eslint/eslint.js

```diff
@@ -43,6 +43,7 @@ const { version } = require("../../package.json");
  * @property {ConfigData} [baseConfig] Base config object, extended by all configs used with this instance
  * @property {boolean} [cache] Enable result caching.
  * @property {string} [cacheLocation] The cache file to use instead of .eslintcache.
+ * @property {"metadata" | "content"} [cacheStrategy] The strategy used to detect changed files.
```

Comment:
This file shouldn't have any Node.js-specific information inside of it. It doesn't directly access the cache, so I think we can safely remove all of the cache-related changes from this file.

Reply:
Oops, please ignore this comment. I misread the filename. This looks good. :+1: